src/main/java/com/denizenscript/depenizen/bukkit/events/mythicmobs/MythicMobsSpawnEvent.java

```diff
@@ -81,6 +87,7 @@ else if (name.equals("spawner_location") && event.isFromMythicSpawner()) {
     public void onMythicMobSpawns(MythicMobSpawnEvent event) {
         mythicmob = new MythicMobsMobTag(event.getMob());
         location = new LocationTag(event.getLocation());
+        reason = event.getSpawnReason();
```

Comment:
This doesn't need to be read in advance, this can be read inside the `getContext` method